lib/play.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './game.rb'
```

Comment:
I'm entirely not sure whether this should be pushed as a part of this story, it seems to be more like a file for you to manually test the logic.

Reply:
Good point. I'll ensure this file is not tracked / removed from the repo